README.md

```diff
@@ -101,6 +103,76 @@ $ docker-compose up
 
 That's it! Just go to <http://localhost:3000>.
 
+## Accessing Recaptcha Variables
+
+Access the https://www.google.com/recaptcha/intro/v3.html.
+
+Click on the `Admin Console` button.
+
+On the new page, click the `+` button and fill in the fields to generate a new site, choose v2 for the captcha type and in domains add `127.0.0.1 and localhost`.
+
+When you click save it will generate your keys.
+
+## Accessing Facebook Variables
+
+Access the https://developers.facebook.com/apps/
+
+Click on the `Add new app` button.
+
+Fill in the name of the app and your contact email. Then click Create Application ID.
+
+On your app page, click on my left side under `Settings -> Basic`.
+
+In this page you need to configure two fields:
+
+- Application domain with `localhost`
+- In the site session the site URL field with `http: // localhost: 3000`, where 3000 is the port used to start the Rails application on your computer.
```

Comment:
I would prefer if instead of `http: // localhost: 3000` we use `http://localhost:3000` format

Reply:
@lucianosousa c52904b8aa89b15d888a90bee0d10a835ba77661